src/operators.jl

```diff
@@ -11,6 +11,8 @@
 ##=================================#
 # Operators for functions f: ℝ → ℝ #
 #==================================#
+function is_firstder_zero_global end
+function is_seconder_zero_global end
```

Comment:
maybe a shorter option
```julia
der1_nonzero
der2_nonzero
der1_arg1_nonzero
der1_arg2_nonzero
der2_arg1_nonzero
der2_arg2_nonzero
der2_arg12_nonzero
```
We could even drop the `nonzero` everywhere since those are internal traits anyway

Reply:
alternative

```julia
has_deri_argj
```

Reply:
Some behind the scenes on the name selection:

I enjoy the Julia convention of boolean functions starting with `is*`. It makes it easy to guess the return type and `if/else` statement read like english sentences.

Since we need to distinguish between locally and globally zero, we might just as well add it the name for the sake of clarity instead of just distinguishing the two through their amount of arguments. 
And since `nonzero_global` doesn't make sense in the context of sparsity, I opted for `zero_global`.

I have no strong feelings about anything between `is` and `zero_global(ly)`.

Reply:
no strong feelings here either, but I would like to replace `first` and `second` by `1` and `2` nonetheless, if only because they are the same length